.circleci/workflows/openlineage-spark.yml

```diff
@@ -120,16 +128,32 @@ workflows:
               ]
           requires:
             - approval-integration-spark
+          post-steps:
```

Comment:
Is this neccessary? Why not stay with previous implementation but add `when: always`?

Reply:
I don't an easy way to solve this. 

The problem is:
 * we do have many jobs
 * if a job does fails, next job is not starting
 * slack notify needs to depend on all the tests' jobs.